src/app/collection/pages/collection-ncyte/feature/collection-learning-object-card/collection-learning-object-card.component.html

```diff
@@ -0,0 +1,33 @@
+<div class = "container">
+
+    <div *ngIf = "learnObj.length !== 'micromodule'">
```

Comment:
The div styling should be according to the learningObject.length. You can use this learningObject card as a guide: https://github.com/Cyber4All/clark-client/blob/master/src/app/cube/shared/learning-object/learning-object.component.html. You can conditionally set classes according to length.

Reply:
it should be changed in the latest commit, just didnt push the commit. But it should be good now I think